modules/contact-form/grunion-contact-form.php

```diff
@@ -2550,7 +2563,10 @@ function render() {
 		$field_label       = $this->get_attribute( 'label' );
 		$field_required    = $this->get_attribute( 'required' );
 		$placeholder       = $this->get_attribute( 'placeholder' );
-		$class             = 'date' === $field_type ? 'jp-contact-form-date' : $this->get_attribute( 'class' );
+    $class             = 'date' === $field_type ? 'jp-contact-form-date' : $this->get_attribute( 'class' );
+		
+		/** This filter is already documented in modules/contact-form/grunion-contact-form.php */
```

Comment:
Sorry, but I think you'll need to bring back the doc block here. I think @dereksmart initially thought that it was the same filter, and it's not. Also please use consistent namespacing. Would `jetpack_contact_form_input_class` work here instead?